File: server/当前版本/server/zlj/game_server/src/com/imop/lj/gameserver/player/msg/GCPlayerChargeDiamond.java
package com.imop.lj.gameserver.player.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 兑换成功
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCPlayerChargeDiamond extends GCMessage{
	
	/** 平台币余额 */
	private int mmBalance;

	public GCPlayerChargeDiamond (){
	}
	
	public GCPlayerChargeDiamond (
			int mmBalance ){
			this.mmBalance = mmBalance;
	}

	@Override
	protected boolean readImpl() {

	// 平台币余额
	int _mmBalance = readInteger();
	//end



		this.mmBalance = _mmBalance;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 平台币余额
	writeInteger(mmBalance);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_PLAYER_CHARGE_DIAMOND;
	}
	
	@Override
	public String getTypeName() {
		return "GC_PLAYER_CHARGE_DIAMOND";
	}

	public int getMmBalance(){
		return mmBalance;
	}
		
	public void setMmBalance(int mmBalance){
		this.mmBalance = mmBalance;
	}
}

File: u3u2/Assets/Scripts/app/net/s2c/GCPlayerChargeDiamond.cs

using System;
namespace app.net
{
/**
 * 兑换成功
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCPlayerChargeDiamond :BaseMessage
{
	/** 平台币余额 */
	private int mmBalance;

	public GCPlayerChargeDiamond ()
	{
	}

	protected override void ReadImpl()
	{
	// 平台币余额
	int _mmBalance = ReadInt();


		this.mmBalance = _mmBalance;
	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_PLAYER_CHARGE_DIAMOND;
	}
	
	public override string getEventType()
	{
		return PlayerGCHandler.GCPlayerChargeDiamondEvent;
	}
	

	public int getMmBalance(){
		return mmBalance;
	}
		

}
}